experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -0,0 +1,1380 @@
+#    set_attribute!(projective_version, :as_covered_scheme, covered_version)
+#    @show "computing a default covering"
+#    set_attribute!(projective_version, :standard_covering, default_covering(covered_version))
+#    @show "done"
+#
+#    proj_dict = Dict{SpecType, morphism_type(SpecType, SpecType)}()
+#    @show "doing some other stuff"
+#    for i in 1:length(I)
+#      @show i
+#      Z = patches(default_covering(covered_version))[i]
+#      U = patches(default_covering(covered_ambient))[i]
+#      proj_dict[Z] = restrict(ambient_projection_map[U], Z, codomain(ambient_projection_map[U]), check=false)
+#    end
+#
+#    projection_map = CoveringMorphism(default_covering(covered_version), Covering(W), proj_dict)
+#    set_attribute!(projective_version, :covered_projection_to_base, projection_map)
+#    @show 3
+#    E_dict = Dict{affine_patch_type(covered_version), ideal_type(ring_type(affine_patch_type(covered_version)))}()
+#    for i in 1:length(I)
+#      @show i
+#      U = default_covering(covered_version)[i]
+#      E_dict[U] = ideal(OO(U), pullback(projection_map[U])(I[i]))
+#    end
+#    @show 4
+#    exc_div = IdealSheaf(covered_version, default_covering(covered_version), E_dict, check=false)
+#    @show "processing done."
+#    return projective_version, covered_version, projection_map, exc_div
+#  end
+#end
+
+function blow_up(
+    I::IdealSheaf;
+    verbose::Bool=false,
+    check::Bool=true,
+  )
+  X = space(I)
+  local_blowups = IdDict{AbsSpec, AbsProjectiveScheme}()
+  for U in affine_charts(X)
+    local_blowups[U] = blow_up(U, I(U))
+  end
+  projective_glueings = IdDict{Tuple{AbsSpec, AbsSpec}, ProjectiveGlueing}()
+
+  # prepare for the projective glueings
+  C = default_covering(X)
+  for (U, V) in keys(glueings(C))
+    P = local_blowups[U]
+    SP = ambient_coordinate_ring(P)
+    Q = local_blowups[V]
+    SQ = ambient_coordinate_ring(Q)
+    G = C[U, V]
+    UV, VU = glueing_domains(G)
+    f, g = glueing_morphisms(G)
+
+    OX = StructureSheafOfRings(X)
+    PUV, PUVtoP = fiber_product(OX(U, UV), P)
+    QVU, QVUtoQ = fiber_product(OX(V, VU), Q)
+
+    # to construct the identifications of PUV with QVU we need to 
+    # express the generators of I(U) in terms of the generators of I(V)
+    # on the overlap U ∩ V. 
+    G isa SimpleGlueing || error("method not implemented for this type of glueing")
+
+    # The problem is that on a SpecOpen U ∩ V
+    # despite I(U)|U ∩ V == I(V)|U ∩ V, we 
+    # have no method to find coefficients aᵢⱼ such that fᵢ = ∑ⱼaᵢⱼ⋅gⱼ
+    # for the generators fᵢ of I(U) and gⱼ of I(V): Even though 
+    # we can do this locally on the patches of a SpecOpen, the result 
+    # is not guaranteed to glue to global functions on the overlap.
+    # Abstractly, we know that the intersection of affine charts 
+    # in a separated scheme must be affine, but we do not have a 
+    # model of this overlap as an affine scheme and hence no computational
+    # backup. 
```

Comment:
I would like to also draw your attention to this here. 

From the top of my head, I had no idea how to resolve this. @simonbrandhorst and I already talked about this a little, but didn't resolve it, yet. The best we could come up with was to do the computations on a dense affine open subset and then try to extend the result. But I see no reason why this would always be possible. 

@jankoboehm : Any ideas on this?